fastlane/Fastfile

```diff
@@ -300,7 +302,7 @@ REPOSITORY_NAME = 'WordPress-Android'
   desc 'Updates a release branch for a new beta release'
   lane :new_beta_release do |options|
     android_betabuild_prechecks(base_version: options[:base_version], skip_confirm: options[:skip_confirm])
-    send_strings_for_translation()
+    update_frozen_strings_for_translation()
```

Comment:
Nitpick. While we're at it:

```suggestion
    update_frozen_strings_for_translation
```

Like my previous suggestion, the rationale is merely adhering to the Rubocop-standard Ruby style.